paparazzi/paparazzi-gradle-plugin/src/test/projects/compose-a11y/src/test/java/app/cash/paparazzi/plugin/test/ComposeA11yTest.kt

```diff
@@ -9,6 +9,7 @@ import org.junit.Test
 class ComposeA11yTest {
   @get:Rule
   val paparazzi = Paparazzi(
+    theme = "Theme.Cash.Accent",
```

Comment:
"Theme.AppCompat.Light.NoActionBar" here instead and remove the "accessibility-insets" test?